src/USDA.NET/Food/Search/SearchResult.cs

```diff
@@ -0,0 +1,29 @@
+using System.Linq;
+using JetBrains.Annotations;
+using Newtonsoft.Json;
+
+namespace USDA.NET.Food.Search
+{
+	public class SearchResult
+	{
+        /// <summary>
+        /// Information about the items returned.
+        /// </summary>
+        [CanBeNull]
+	    [JsonProperty("list")]
+        public SearchList List { get; set; }
+
+        /// <summary>
+        /// List of errors.
+        /// </summary>
+        [JsonProperty("errors")]
+	    public Errors Errors { get; set; } = new Errors();
```

Comment:
This default property behavior is inconsistent (why is `Errors` special?) and unnecessary (what value is gained from this being a non-null but invalid object?).

Reply:
because it's easier to do Errors.Any() versus. Errors != null && Errors.Any()